docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,156 @@
+---
+lang: en
+title: 'Inside a Loopback Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-Loopback-Application.html
+---
+
+# APIs in a digital era
+
+Digital transformation is changing how data and services are shared across Web/Mobile/IoT platforms.
+APIs allow easy and secure access to digital resources and capabilities of an organization by internal and enternal parties.
+
+Developers are looking out for more and more productive ways of creating and consuming APIs.
+
+Consider a typical `Online Shopping` site. First the UI requirements are gathered. Customers would be provided a home page, a product list, a shopping cart and an order history page.
+
+Next, the API requirements are assessed. If the capability to choose products, add to cart and submit an order are exposed as secure APIs, they can be used by an in-house web application as well as a third party mobile app developer. 
+
+Once the API requirements are assessed,
+
+- Front end designers would want to try the `Online Shopping` UI experience.
+  > Loopback provides options to configure simple methods as remote API endpoints.
+
+- API developers would want to assess API specifications and security options.
+  > Loopback provides options to decorate remote methods with security options.
+
+- Back end developers would want to create Entity-Relationship models from backend datasource
+  > Loopback provides extensive features to connect to backend and generate object models from datasource schemas.
+
+The example application [loopback4-example-shopping](https://github.com/strongloop/loopback4-example-shopping)
+is a nice place for developers to start looking on how to use LoopBack 4.
+
+In the example, `Redis` is used for caching items in the shopping cart and `MongoDB` is used for saving user and order data.
```

Comment:
Let's add a diagram to show backend realities.

Reply:
I have added the image from example-shopping readme in here, hope that should suffice. or do we need a more detailed one ?